module/generator/src/Printer/ErgoPrinter.php

```diff
@@ -17,11 +17,15 @@
 class ErgoPrinter extends Printer
 {
     /**
+     * {@inheritdoc}
+     *
      * @var string
      */
     protected $indentation = '    ';
 
     /**
+     * {@inheritdoc}
+     *
      * @var int
      */
     protected $linesBetweenMethods = 1;
```

Comment:
Why whithout type?

Reply:
It's overriding parent property therefore PHP disallows declaring the type as it's not available in a parent clas.